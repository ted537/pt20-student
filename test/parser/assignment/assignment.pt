using input, output;
var a = 5;
var b = 0;
b = a;
