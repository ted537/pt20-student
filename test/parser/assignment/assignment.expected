 .sProgram
   .sIdentifier
   .sIdentifier
  % .sNewLine
  .sParmEnd
  .sBegin
    .sVar
     .sIdentifier
     .sInitialValue
         .sInteger
         .sExpnEnd
    % .sNewLine
    .sAssignmentStmt
    .sIdentifier
        .sInteger
    .sExpnEnd
    % .sNewLine
  .sEnd
